packages/flutter_tools/lib/src/commands/attach.dart

```diff
@@ -104,6 +104,10 @@ class AttachCommand extends FlutterCommand {
         'debug-url',
         aliases: <String>[ 'debug-uri' ], // supported for historical reasons
         help: 'The URL at which the observatory is listening.',
+      )..addFlag('serve-observatory',
```

Comment:
nit: can you move the definition of this flag a helper method in https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/runner/flutter_command.dart#L297, so that if we (for example) change the help text they all stay synched?